crates/oxc_allocator/src/vec.rs

```diff
@@ -35,7 +34,7 @@ use crate::{Allocator, Box};
 /// Static checks make this impossible to do. [`Vec::new_in`] and all other methods which create
 /// a [`Vec`] will refuse to compile if called with a [`Drop`] type.
 #[derive(PartialEq, Eq)]
-pub struct Vec<'alloc, T>(pub(crate) ManuallyDrop<InnerVec<'alloc, T>>);
+pub struct Vec<'alloc, T>(pub(crate) InnerVec<'alloc, T>);
```

Comment:
I don't think `pub(crate)` is required any more either.

```suggestion
pub struct Vec<'alloc, T>(InnerVec<'alloc, T>);
```